desktop/core/src/desktop/hue_shared.py

```diff
@@ -0,0 +1,24 @@
+# Licensed to Cloudera, Inc. under one
```

Comment:
This is duplicated with https://github.com/cloudera/hue/blob/master/tools/ops/script_runner/lib/hue_shared.py? Maybe update the import statement?